source/docs/articles/wordpress/cloudFront-setup-for-wordpress.md

```diff
@@ -62,7 +62,7 @@ There are a few different ways you can install a plugin on a Pantheon hosted Wor
 Execute the following [Terminus](/docs/articles/local/cli/) command to install and activate the Amazon S3 and CloudFront plugin:
 
 ```bash
-terminus wp plugin install amazon-s3-and-cloudfront --activate --site=your-awesome-site --env=dev
+terminus wp 'plugin install amazon-s3-and-cloudfront' --activate --site=your-awesome-site --env=dev
```

Comment:
@nataliejeremy closing quote too early, needs to enclose the `--activate` option. Check all of the commands for this. Only `--site=<site> --env=<env>` should be outside of the quotes, after the `terminus wp` command.